cmake/modules/FindPULSEAUDIO.cmake

```diff
@@ -0,0 +1,120 @@
+#.rst:
```

Comment:
Add license info found in https://invent.kde.org/frameworks/extra-cmake-modules/-/blob/master/find-modules/FindPulseAudio.cmake

and copy and paste the text of the license
https://invent.kde.org/frameworks/extra-cmake-modules/-/blob/master/LICENSES/BSD-3-Clause.txt

Reply:
oh, I see that is below. Please move it to the top of the file.